src/meta/src/barrier/mod.rs

```diff
@@ -909,6 +912,10 @@ where
                             tracker.update(&progress);
                         }
                     }
+                } else if prev_epoch != INVALID_EPOCH {
+                    self.hummock_manager
+                        .update_current_epoch(prev_epoch)
```

Comment:
I think we also need to update current epoch when the barrier is a checkpoint barrier.

Reply:
ignore this comment. just realize that current epoch will be updated on `commit_epoch` implementation.